### PR TITLE
fix: fixes incorrect date formatting

### PR DESCRIPTION
**Pull request by jeancallisti**
Before : 2022-4-19T1372
After:    2022-04-19T13:**0**7:**0**2

### CODE REVIEW

**Review by StraToN — APPROVED** [review]

File: addons/escoria-core/game/core-scripts/log/esc_logger.gd [hunk 1]
```diff
@@ -262,17 +262,29 @@ func report_errors(p_path: String, errors: Array) -> void:
...
 	else:
 		print(message)
 	_write_logfile(message)
 
 
+# Returns the current date/time, as a string 
+# formatted for logging. E.g. 2022-04-19T16:10:39
+func _formatted_date() -> String:
+	var info = OS.get_datetime()
+	info["year"] 	= "%04d" % info["year"]
```
Comment (StraToN) [inline addons/escoria-core/game/core-scripts/log/esc_logger.gd:278]: Please remove tabs in the middle of lines.

**Review by StraToN — CHANGES_REQUESTED** [review]

File: addons/escoria-core/game/core-scripts/log/esc_logger.gd (hunk 1 above)
Comment (jeancallisti) [reply addons/escoria-core/game/core-scripts/log/esc_logger.gd:278]: https://stackoverflow.com/questions/101958/code-formatting-is-lining-up-similar-lines-ok

Comment (balloonpopper) [reply addons/escoria-core/game/core-scripts/log/esc_logger.gd]: Can you please add some context to your response rather than just a URL? @jeancallisti

Comment (jeancallisti) [reply addons/escoria-core/game/core-scripts/log/esc_logger.gd]: I applied the requested fix anyways, so let's just move forward

**Review by StraToN — APPROVED** [review]

**Review by BHSDuncan — APPROVED** [review]

**Pull request merged**